### DISCUSSION TITLE
"unknown kind: btfKind(19)" in 5.14.0-229.el9.x86_64(centos stream) enviroment

### DISCUSSION THREAD

**Discussion by zhangzju** [cilium/ebpf#900] [Q&A] [answered]
I try to run my program in a centos stream virtual machine with kernel 5.14.0-229.el9.x86_64, throw an error as below:

```bash
apply CO-RE relocations: can't read types: type id 4124: unknown kind: btfKind(19)
```
the version of ebpf in my go.mod is v0.9.3 , I found that this problem seems solved in master branch now , 
```go
		switch header.Kind() {
/*...*/
		case kindTypeTag:
		case kindEnum64:
			data = make([]btfEnum64, header.Vlen())
		default:
			return nil, fmt.Errorf("type id %v: unknown kind: %v", id, header.Kind())
		}
```
so I solve the problem temporarily via this way:

1. go vendor.
2. remove the ebpf dir and clone the latest code.
3. generate ebpf program and build again.

It worked now, but not ready for production, the v0.9.3 release is the latest release now, may I ask when the new version will release?

**Comment by lmb** [comment, marked as answer]
We're going to cut a new release next week most likely. The steps you took to use the latest version of the package are... unorthodox :) Next time just running `go get github.com/cilium/ebpf@master` should suffice.

**Reply by zhangzju** [reply]
Thanks, it works for me!
